docs/source/user_guide/fft.rst

```diff
@@ -16,6 +16,49 @@ or may be absent in `hipFFT`_/`rocFFT`_ targeting AMD GPUs.
 .. _rocFFT: https://rocfft.readthedocs.io/en/latest/
 
 
+.. _scipy_fft_backend:
+
+SciPy FFT backend
+-----------------
+
+Since SciPy v1.4 a backend mechanism is provided so that users can register different FFT backends and use SciPy's API to perform the actual transform
+with the target backend, such as CuPy's :mod:`cupyx.scipy.fft` module. For a one-time only usage, a context manager :func:`scipy.fft.set_backend` can be used:
+
+.. code-block:: python
+
+    import cupy as cp
+    import cupyx.scipy.fft as cufft
+    import scipy.fft
+
+    a = cp.random.random(100).astype(cp.complex64)
+    with scipy.fft.set_backend(cufft):
+        b = scipy.fft.fft(a)  # equivalent to cufft.fft(a)
+
+However, such usage can be tedious. Alternatively, users can register a backend through :func:`scipy.fft.register_backend` or :func:`scipy.fft.set_global_backend`
+to avoid using context managers:
+
+.. code-block:: python
+
+    import cupy as cp
+    import cupyx.scipy.fft as cufft
+    import scipy.fft
+    scipy.fft.set_global_backend(cufft)
+
+    a = cp.random.random(100).astype(cp.complex64)
+    b = scipy.fft.fft(a)  # equivalent to cufft.fft(a)
+
+.. note::
+
+    Currently in case of conflicts the precedence of the three APIs is :func:`~scipy.fft.set_backend` > :func:`~scipy.fft.set_global_backend`
```

Comment:
This order is true in general but might not be true if the user decides to use `try_last=True`. Should we mention that somewhere? @peterbell10 @leofang

Reply:
Thanks, @AnirudhDagar! Given that in the next sentence I referred to the SciPy documentation, I think this sentence should just be removed. It's SciPy's responsibility to explain implementation details such as the precedence, not CuPy's. If it'd be ever changed one day our docs would be outdated.

Reply:
Yeah, this sounds like a better idea.